plugin.video.worldstarhiphop/resources/settings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
You might want to convert this to the new settings format. There's an addon you can use to do that (not mandatory): https://gitlab.com/ronie/script.settings-convert

Reply:
Done